src/lerobot/configs/policies.py

```diff
@@ -89,7 +94,7 @@ def __post_init__(self):
 
     @property
     def type(self) -> str:
-        return self.get_choice_name(self.__class__)
+        return cast(str, self.get_choice_name(self.__class__))
```

Comment:
`cast` should be always justified in the code, this is unsafe. 
In this case it should be ok.